Balance/BalanceAppDelegate.swift

```diff
@@ -35,8 +35,6 @@ class BalanceAppDelegate: CardinalKitAppDelegate {
             if HKHealthStore.isHealthDataAvailable() {
                 healthKit
             }
-#else
-            FirebaseAccountConfiguration()
```

Comment:
@gperissetcelteeka just checking we don't need this.

Reply:
Its ok